.github/workflows/ruby.yml

```diff
@@ -1,6 +1,6 @@
 # License: LGPL-3.0-or-later
 name: Ruby test and build
-on: 
+on:
```

Comment:
Make sure to remove the reformatting here. There's nothing really wrong with the end result but it clutters the PR